src/DynamoCoreWpf/UI/Themes/Modern/Ports.xaml

```diff
@@ -88,19 +87,13 @@
                 <Rectangle
                     Fill="Transparent"
                     SnapsToDevicePixels="True"
-                    IsHitTestVisible="{Binding IsHitTestVisible}">
+                    IsHitTestVisible="{Binding IsHitTestVisible}"
+                        views:EventToCommand.Events="MouseEnter,MouseLeave,MouseLeftButtonDown"
```

Comment:
note that now we pass a list of event names and command names  - we can do some validation that these need to have the same number of entries at runtime if we want.